src/main/groovy/com/ullink/gradle/opencover/OpenCover.groovy

```diff
@@ -26,6 +27,8 @@ class OpenCover extends ConventionTask {
     boolean mergeOutput = false
     boolean skipAutoProps = false
 
+    def fileNameId = new AtomicLong(1)
```

Comment:
Shouldn't this be static? :D

Reply:
if there are multiple opencover tasks defined, probably yes.

Reply:
Fixed in https://github.com/Ullink/gradle-opencover-plugin/pull/24